Goobi/src/de/sub/goobi/metadaten/RenderableMetadataGroup.java

```diff
@@ -157,6 +157,34 @@ public RenderableMetadataGroup(MetadataGroup data, Metadaten container, String l
         setLanguage(language);
     }
 
+    /**
+     * Creates a RenderableMetadataGroup instance for a default display metadata
+     * group that still can be added to the currently selected level of the
+     * document structure hierarchy.
+     *
+     * @param type
+     *            type of metadata group
+     * @param language
+     *            display language to use
+     * @param projectName
+     *            project that the act whose metadata group is to edit belongs
+     *            to
+     * @throws ConfigurationException
```

Comment:
why is it not possible to show a single selection in a multi-select?

Reply:
You have misread. There must be no **multiple** selection in the metadata group.